services/spar/test-integration/Test/Spar/ScimSpec.hs

```diff
@@ -139,61 +144,113 @@ specUsers = describe "operations with users" $ do
     describe "PUT /Users" $ do
         it "responds with 4xx (just making sure...)" $ do
             env <- ask
-            user <- randomSCIMUser
-            (tok, _) <- registerIdPAndSCIMToken
-            putUser_ (Just tok) Nothing user (env ^. teSpar)
+            user <- randomScimUser
+            (tok, _) <- registerIdPAndScimToken
+            updateUser_ (Just tok) Nothing user (env ^. teSpar)
                 !!! assertTrue_ (inRange (400, 499) . statusCode)
 
     describe "PUT /Users/:id" $ do
         it "updates the user attributes in scim_user" $ do
-            env <- ask
-            user <- randomSCIMUser
-            (tok, _) <- registerIdPAndSCIMToken
+            -- Create a user via SCIM
+            user <- randomScimUser
+            (tok, _) <- registerIdPAndScimToken
             storedUser <- createUser tok user
             let userid = scimUserId storedUser
-            user' <- randomSCIMUser
```

Comment:
I wrote a wrapper for `putUser_` because all other raw functions also have wrappers. Now we have `updateUser` and `updateUser_`.